manifests/ssltools/nssdb.pp

```diff
@@ -0,0 +1,42 @@
+# Sets up nssdb
+class certs::ssltools::nssdb (
+  $nss_db_dir = $::certs::nss_db_dir,
+  $qpidd_group = $::certs::qpidd_group
```

Comment:
Given this is now just about creating the nss db I think it could even be just ```$group = ...``` and use a trailing comma.